lib/resources/example_json/dstu2_examples_slot.rb

```diff
@@ -286,5 +286,463 @@ module Resources
+            },
+            "extension": [
+              {
+                "url": "https://fhir-ehr.cerner.com/dstu2/StructureDefinition/scheduling-location",
+                "valueReference": {
+                  "reference": "Location/4048128"
+                }
+              }
+            ],
+            "type": {
+              "coding": [
+                {
+                  "system": "http://snomed.info/sct",
+                  "code": "394581000",
+                  "display": "Community medicine",
+                  "userSelected": false
+                },
+                {
+                  "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/14249",
+                  "code": "21265426",
+                  "display": "Same Day",
+                  "userSelected": true
+                }
+              ],
+              "text": "Same Day"
+            },
+            "schedule": {
+              "reference": "Schedule/21265426-4048128-3121785-5"
+            },
+            "freeBusyType": "free",
+            "start": "2016-01-31T14:05:00.000Z",
+            "end": "2016-01-31T15:05:00.000Z"
+          },
+          "search": {
+            "mode": "match"
+          }
+        },
+        {
+          "fullUrl": "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Schedule/21265426-4048128-3121781-420",
+          "resource": {
+            "resourceType": "Schedule",
+            "id": "21265426-4048128-3121781-420",
+            "text": {
+              "status": "extensions",
+              "div": "&lt;div&gt;&lt;p&gt;&lt;b&gt;Schedule&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Type&lt;/b&gt;: Same Day&lt;/p&gt;&lt;p&gt;&lt;b&gt;Practitioner&lt;/b&gt;: Howdeshell, Tami&lt;/p&gt;&lt;p&gt;&lt;b&gt;Location&lt;/b&gt;: Baseline East&lt;/p&gt;&lt;/div&gt;"
+            },
+            "extension": [
+              {
+                "url": "https://fhir-ehr.cerner.com/dstu2/StructureDefinition/scheduling-location",
+                "valueReference": {
+                  "reference": "Location/4048128",
+                  "display": "Baseline East"
+                }
+              }
+            ],
+            "type": [
+              {
+                "coding": [
+                  {
+                    "system": "http://snomed.info/sct",
+                    "code": "394581000",
+                    "display": "Community medicine"
+                  }
+                ],
+                "text": "Same Day"
+              }
+            ],
+            "actor": {
+              "reference": "Practitioner/2578010",
+              "display": "Howdeshell, Tami"
```

Comment:
Replace all of these displays with a made up person (and in the text.div field

Reply:
https://github.com/cerner/fhir.cerner.com/pull/256/commits/42269d94e855b35f116fba9fccc9feff6eada449

Reply:
Oops, accidentally updated the Pract ID too. Reverted https://github.com/cerner/fhir.cerner.com/pull/256/commits/7d79cb17af18d809a419eb3dfb0ddc316adcbd5a